src/jobs/index.ts

```diff
@@ -54,7 +52,21 @@ export async function handleJobRoute(
   } catch (e) {
     reply.code(401);
     reply.send();
-    return;
+    return false;
+  }
+  return true;
+}
+
+// Error handling wrapper function for Cron Jobs involving Github
+// Additionally handles Auth from Cloud Scheduler
+export async function handleGithubJobs(
```

Comment:
Nit: Could we use singular for the naming here? Also, GitHub is usually written with a capital `H`
```suggestion
export async function handleGitHubJob(
```